src/gleam_stdlib.erl

```diff
@@ -363,14 +369,23 @@ inspect(true) ->
     "True";
 inspect(false) ->
     "False";
+inspect(nil) ->
+    "Nil";
 inspect(Any) when is_atom(Any) ->
-    lists:map(
-        fun(Part) ->
-            [Head | Tail] = string:next_grapheme(unicode:characters_to_binary(Part)),
-            [string:uppercase([Head]), Tail]
-        end,
-        re:split(erlang:atom_to_list(Any), "_+", [{return, iodata}])
-    );
+    AtomAsList = erlang:atom_to_list(Any),
+    case inspect_maybe_gleam_atom(AtomAsList, none, []) of
+        {ok, GleamCompatibleAtomString} -> erlang:list_to_binary(GleamCompatibleAtomString);
+        {error, Reason} -> case Reason of
```

Comment:
All these atoms should be shown the same way so there's no need to return a different error and match on each one. Can just be `{error, nil}`

Reply:
I can collapse some of the errors but there are 3 outcomes:
- ´["//erl('", erlang:atom_to_binary(Any), "')"]´ -- standard case
- ´["//erl('')"]´ -- empty string
- ´["//erl(", erlang:atom_to_binary(Any), ")"]´ How Erlang represents atoms with double underscores or trailing underscores.

Reply:
Always the first case please